scripts/global.lua

```diff
@@ -65,7 +65,9 @@ function onAfterLoad()
    loadData()
    fhLogSettingsUpdated()
    local settings = JSON.decode(getSettings())
-   updateHotkeys({ enabled = settings["enable-solo"] or false, fivePlayers = settings["enable-5p"] or false })
+   local params = { enabled = settings["enable-solo"] or false, fivePlayers = settings["enable-5p"] or false }
+   updateHotkeys(params)
+   getScenarioMat().call('updateSnapPoints', params)
```

Comment:
Can you please comment this line out, I will update the save instead. This avoids a reload of the scenario mat.

Reply:
Should be done!